segway/run.py

```diff
@@ -3944,11 +3944,68 @@ def parse_options(argv):
     identify_finish = tasks.add_parser("", add_help=False)
     identify_init_run = tasks.add_parser("", add_help=False)
 
+    # Common options across tasks
+    class Argument(object):
+        def __init__(self, args, kwargs):
```

Comment:
I would `def __init__(self, *args, **kwargs):`. Then you can do things like

```
include_coords_args = Argument("--include-coords",
                               metavar="FILE", help="limit to genomic coordinates)
```

where the interface is identical between use here or passing directly to `argparse.ArgumentParser.add_argument()`